packages/aws-cdk/lib/api/bootstrap/bootstrap-template.yaml

```diff
@@ -129,6 +129,23 @@ Resources:
         IgnorePublicAcls: true
         RestrictPublicBuckets: true
     UpdateReplacePolicy: Retain
+  StagingBucketPolicy:
+    Type: 'AWS::S3::BucketPolicy'
+    Properties:
+      Bucket: { Ref: 'StagingBucket' }
+      PolicyDocument:
+        Id: 'AccessControl'
+        Version: '2012-10-17'
+        Statement:
+          - Sid: 'AllowSSLRequestsOnly'
+            Action: 's3:*'
+            Effect: 'Deny'
+            Resource:
+              - { 'Fn::Sub': '${StagingBucket.Arn}' }
+              - { 'Fn::Sub': '${StagingBucket.Arn}/*' }
```

Comment:
Could this not just be `*`? This is a `Deny` within a `BucketPolicy`, so I don't see what being so specific adds... beyond maybe a risk of inadvertently not including something?

Reply:
I guess. This is just the way the recommended policy looks.

Reply:
Well then!  This is unironically...

![](https://images.squarespace-cdn.com/content/v1/5574740be4b092aac3812677/1492096676567-47XPDZ29M7ON2DIQ1KE7/ke17ZwdGBToddI8pDm48kAolcJLata8vUmgOJDHjSf17gQa3H78H3Y0txjaiv_0fDoOvxcdMmMKkDsyUqMSsMWxHk725yiiHCCLfrh8O1z4YTzHvnKhyp6Da-NYroOW3ZGjoBKy3azqku80C789l0luUmcNM2NMBIHLdYyXL-Jyjz76XRJzHmys88oCWP7kZRqO_7ozNDIra3IXZlg8J3Q/goodenough_8x11.JPG)